node/get_price/README.md

```diff
@@ -0,0 +1,55 @@
+# 📷 Get Price Function
+
+A Node Cloud Function for getting the price of crypto, stocks, and commodities. Supported goods include (at least) Bitcoin, Ethereum, Google, Amazon, gold, and silver.
+
+_Example input:_
+
+```json
+{
+    "type": "gold"
+}
+```
+
+_Example output:_
+
+
+```json
+{
+    "success":true,
+    "price":1697.6
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Please remove references to "environment" as those were in the older version of functions.